### DISCUSSION TITLE
A button to switch between text path and buttons path in file open menu window

### DISCUSSION THREAD

**Discussion by santoshicha** [unixwork/xnedit#119] [Ideas]
In original NEdit, the file open window has path in text format and in XNEdit has the buttons for path.
If I need to switch path in 2 different root path but same hierarchy, text format path is easy.

Thank you.

**Comment by unixwork** [comment]
You can click in the white background next to the buttons and it switches to text mode.

**Reply by santoshicha** [reply]
Thank you.
